Sources/App/Views/Sections/Editor/SourceTree/SourceTreeViewModel.swift

```diff
@@ -99,11 +111,7 @@ final class SourceTreeViewModel: ObservableObject {
 
   /// Updates the `directoryContent` by iterating over the contents of the workspace directory.
   func refreshContent() throws {
-    guard let workspaceDirectory = workspaceURL else {
-      throw MockaError.missingWorkspacePathValue
-    }
-
-    directoryContent = Logic.SourceTree.contents(of: workspaceDirectory)
+    sourceTree = Logic.SourceTree.sourceTree()
```

Comment:
Does updating sourceTree here in this way ensure that the sourceTree in EditorEnvironment is also updated? or is it not relevant?

Reply:
good catch